tests/tests_pytorch/run_standalone_tasks.sh

```diff
@@ -31,9 +31,20 @@ if [ -z "$LOGS" ]; then
     exit 1
 fi
 
+cd ../../
+
 # test that a user can manually launch individual processes
 echo "Running manual ddp launch test"
 export PYTHONPATH="${PYTHONPATH}:$(pwd)"
 args="--trainer.accelerator gpu --trainer.devices 2 --trainer.strategy ddp --trainer.max_epochs=1 --trainer.limit_train_batches=1 --trainer.limit_val_batches=1 --trainer.limit_test_batches=1"
-MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=1 python ../../examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args} &
-MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=0 python ../../examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args}
+MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=1 python examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args} &
+MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=0 python examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args}
+
+# test that ddp can launched as a module (-m option)
+echo "Running ddp example as module"
+#export PYTHONPATH="${PYTHONPATH}:$(python -c "import pytorch_lightning; print(pytorch_lightning.examples.convert_from_pt_to_pl.image_classifier_5_lightning_datamodule.__file__)")"
+export PYTHONPATH="$(pwd)/examples:${PYTHONPATH}"
+export PYTHONPATH="$(pwd)/examples/convert_from_pt_to_pl:${PYTHONPATH}"
+
+echo $PYTHONPATH
+python -m examples.convert_from_pt_to_pl.image_classifier_5_lightning_datamodule ${args}
```

Comment:
Anyone know how to run "python -m" in the CI here?

It can't find the examples despite the exact path matching
```
/usr/bin/python: Error while finding module specification for 'examples.convert_from_pt_to_pl.image_classifier_5_lightning_datamodule' (ModuleNotFoundError: No module named 'examples.convert_from_pt_to_pl')
```

And this nonsense works fine locally for me
Tried messing with the PYTHONPATH to trick it, but no success

Reply:
Maybe it's because the script is run inside the `tests_pytorch` directory, not the repository root: https://github.com/Lightning-AI/lightning/blob/58014846ee0fb54b92e4bfb4c0965b72bc0a9641/.azure/gpu-tests.yml#L123

Reply:
Maybe it's because the `examples` and `convert_from_pt_to_pl` directories do not define an `__init__.py`?

It works locally for me, but there might be PyCharm magic going on.

Note that we use `python -m` in the standalone file, so it should work: https://github.com/Lightning-AI/lightning/blob/58014846ee0fb54b92e4bfb4c0965b72bc0a9641/tests/tests_pytorch/run_standalone_tests.sh#L25

Reply:
Thanks this looks like it worked. 

Do we prefer adding the init files in examples or have a dedicated small script in the test folder like we had before?

Reply:
I am against using examples as tests. So I prefer a dedicated small script.